deploy-0chain/action.yml

```diff
@@ -447,6 +447,28 @@ runs:
           echo "==========================================================="
         fi
 
+    - name: "Install Zs3Server"
+      shell: 'script --return --quiet --command "bash {0}"'
+      run: |
+        ./zwallet faucet --methodName "pour" --tokens 9 --input "{}" --silent
+        ./zbox newallocation --lock 5 --data 2 --parity 2 --silent | grep -o "[a-z0-9]\{64\}" > allocation.txt
+        echo "Allocation hash: $(cat allocation.txt)"
```

Comment:
remove?

Reply:
It's necessary, because allocation is needed in next step